### PR TITLE
Multipart examples update

### PR DESCRIPTION
**Pull request by romain-grecourt**
Add a unit test that uses WebClient in the multipart example for SE.
Add a new example under microprofile that demonstrates how to use `MultiPartFeature`.

### CODE REVIEW

File: examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java [hunk 1]
```diff
@@ -125,41 +107,22 @@ private void streamUpload(ServerRequest req, ServerResponse res) {
...
-        } catch (IOException ex) {
-            throw new RuntimeException(ex);
-        }
-    }
-
-    private static void writeBytes(Path storage, String fname, byte[] bytes) {
+    private static void writeBytes(Path file, byte[] bytes) {
         try {
-            Files.write(storage.resolve(fname), bytes,
+            Files.write(file, bytes,
```
Comment (tomas-langer) [inline examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java:120]: This is a blocking operation that is not allowed within a reactive server processing.

**Review by tomas-langer — CHANGES_REQUESTED** [review]

File: examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java (hunk 1 above)
Comment (romain-grecourt) [reply examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java:120]: The only alternative is to create a dedicated thread-pool, shall I do that  ?

Comment (tomas-langer) [reply examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java:120]: Please create a follow up issue - we need to resolve this for other examples as well.

Comment (romain-grecourt) [reply examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java:120]: Yes and there is also the blocking start of the webserver in the unit test, that you pointed out in the new multiport PR. I copied that from the quickstarts, so I will create another issue to address that.

Comment (romain-grecourt) [reply examples/media/multipart/src/main/java/io/helidon/examples/media/multipart/FileService.java:120]: created #2844 and #2845

**Pull request merged**